src/pages/my_page/user_profile/UserRoles.tsx

```diff
@@ -77,7 +77,8 @@ const RoleItem = ({ dataTestId, label, text }: IconLabelTextLineProps) => (
   <Box
     data-testid={dataTestId}
     sx={{
-      pt: '0.8rem',
+      bgcolor: '#d3c9af',
```

Comment:
Etter en pull fra develop kan vi bruke theme her :D 
```suggestion
      bgcolor: 'secondary.dark',
```